test/e2e/framework/fixture.go

```diff
@@ -258,19 +263,84 @@ func NewWorkspaceWithWorkloads(t *testing.T, server RunningServer, orgClusterNam
 	return orgClusterName.Join(ws.Name)
 }
 
-// StartWorkspaceSyncer starts a new syncer to synchronize the identified set of
-// resource types scheduled for the given workload cluster from the upstream server
-// to the downstream server.
-func StartWorkspaceSyncer(
+// SyncerFixture contains the information to run a syncer fixture.
+type SyncerFixture struct {
+	RunningServer       RunningServer
+	KubeClient          kubernetes.Interface
+	upstreamConfig      *rest.Config
+	downstreamConfig    *rest.Config
+	resources           sets.String
+	logicalClusterName  logicalcluster.LogicalCluster
```

Comment:
Maybe s/logicalClusterName/orgClusterName/? I think that's what it was originally, I can't recall what prompted the name change. Seeing it after some time has passed I don't think `logicalClusterName` is descriptive enough.